public/components/judgment_create/judgment_create.tsx

```diff
@@ -266,7 +277,11 @@ export const JudgmentCreate: React.FC<JudgmentCreateProps> = ({ http, notificati
                   />
                 </EuiCompressedFormRow>
 
-                <EuiCompressedFormRow label="Size" fullWidth>
+                <EuiCompressedFormRow 
+                  label="K Value"
+                  helpText="The depth in the result set to include in judging process."
```

Comment:
Suggestion:
"The number of documents from the result list to include in the judging process."